cms/models.py

```diff
@@ -607,7 +705,55 @@ def program_page(self):
         """
         Gets the program page associated with this course, if it exists
         """
-        return self.course.program.page if self.course and self.course.program else None
+        return self.course.program.page if self.course.program.page else None
```

Comment:
This needs to be changed. As written, it will not work for courses with no program. Change to:

`return self.course.program.page if self.course.program else None`

Reply:
Updated